packages/form-js-viewer/src/features/expression-language/ConditionChecker.js

```diff
@@ -81,33 +81,6 @@ export class ConditionChecker {
     return result === true;
   }
 
-  /**
-   * Evaluate an expression.
-   *
-   * @param {string} expression
-   * @param {import('../types').Data} [data]
-   *
-   * @returns {any}
-   */
-  evaluate(expression, data = {}) {
```

Comment:
Is this technical a breaking change? as `form.get('conditionChecker').evaluate(expression)` would break now.

We could mitigate this by keeping the function, but executing `expressionLanguage.evaluate` under the hood. We could also say it was never public API. Wdyt?

Reply:
Well, I doubt it'll affect many people since we don't have a documented public API, but yeah I guess technically a breaking change. But I also think we can say that we never 'officially' supported it. Then again do we ever officially support anything :trollface: . Might be worth having some kind of a 'contract', maybe API documentation. To know what we can and can't change in the future. Well whenever time permits.